feature/notification/src/main/java/com/iteneum/notification/domain/models/NotificationDomain.kt

```diff
@@ -1,6 +1,7 @@
 package com.iteneum.notification.domain.models
 
 data class NotificationDomain(
+    val typeNotification: Int,
     val date : String,
     val body : String
```

Comment:
We can add a new param for example "headerTitle"

Reply:
yes could be, but for this view model and with this version is not necesary